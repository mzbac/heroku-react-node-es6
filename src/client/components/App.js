import React, { PropTypes } from 'react';
import Card from './Card';
import styles from './App.css';


const App = props => {

  return (
    <div className={styles.container} >
      App
      <Card />
    </div>
  );
};

App.propTypes = {};

export default App;

