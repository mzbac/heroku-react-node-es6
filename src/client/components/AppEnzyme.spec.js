import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import App from './App';

it('shallow renders App correctly', () => {
  const component = shallow(
    <App />
  );
  const tree = shallowToJson(component);
  expect(tree).toMatchSnapshot();
});
