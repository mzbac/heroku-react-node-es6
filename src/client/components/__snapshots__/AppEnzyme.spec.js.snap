exports[`test shallow renders App correctly 1`] = `
<div>
  App
  <Card />
</div>
`;
